EIPS/eip-5732.md

```diff
@@ -10,44 +10,37 @@ type: Standards Track
 category: ERC
 created: 2022-09-29
 requires: 165
+
```

Comment:
I suspect this line 13 adding an empty line before it is breaking the EIPW

Reply:
fixed in https://github.com/fulldecent/EIPs/commit/c3823d281c7b6ec30768665c6742701b1fb8204a